acq4/devices/PatchPipette/states/clean.py

```diff
@@ -19,9 +19,6 @@ class CleanState(PatchPipetteState):
     rinseSequence : list
         List of (pressure (Pa), duration (s)) pairs specifying how to pulse pressure while the pipette tip is in the
         rinse well.
-    approachHeight : float
```

Comment:
We still need approachHeight.. move it to Pipette's config if it doesn't work here.